handler/transport/probe_map/probe_map.go

```diff
@@ -0,0 +1,30 @@
+package probe_map
+
+import (
+	"sync"
+)
+
+type BackendProbes map[string]string
```

Comment:
Maybe you can take a look at the `sync.Map` implementation which could be a lot faster for many reads and less writes. This map variable could be moved next to `Probe` instead of an own package.

Reply:
Implemented a working version using `sync.Map`, however for the sake of avoiding import cycles the map was declared in a not completely suitable package. I would like to go over where to move this map in tomorrows meeting.